modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -760,4 +760,25 @@ definition {
 			locator1 = "PRMMDFRequest#STATUS_BUTTON");
 	}
 
+	@summary = "Assert the value for Budget MDF Request"
```

Comment:
Hi @Italo-gabriel, this summary could be something like 'Assert the Total Cost value of the budget, inside the activities page of the MDF Request'

Reply:
Done !